dsgrid_project/build_registry/README.md

```diff
@@ -0,0 +1,59 @@
+# Build StandardScenarios registry
+The `build_registry.sh` script in this directory builds a dsgrid registry containing all
+StandardScenarios datasets and derived datasets. dsgrid developers need to keep the scripts updated
+as new derived datasets are added.
+
+## Dependencies
+The scripts rely on Apache Spark cluster creation scripts in
+https://github.com/NREL/HPC/tree/master/applications/spark. You must clone it locally.
+```
+$ git clone https://github.com/NREL/HPC
+```
+
+## Preparation
+The script will generate a large amount of data (currently 8-9 TiB). Run the script from
+`/scratch/$USER`. Ensure that the data is correct before you copy it to `/projects/dsgrid` or AWS.
+
+The rest of these instructions assume that you are working from an empty directory in
+`/scratch/$USER`.
+
+Edit `${HOME}/.dsgrid.json5` to point to the correct database and ensure that ``offline`` is true.
+If you don't have the file, run this command with correct values:
+```
+$ dsgrid config create --database-name TEXT --database-url TEXT --offline
+```
+
+### Required modifications
+You must make several changes to `./project_registration.json5` before running the workflow.
+```
+$ cp $DSGRID_SS_REPO_BASE_PATH/dsgrid_project//build_registry/project_registration.json5 .
+```
+Open `project_registration.json5` with a text editor and make these changes:
+- Modify the `conn` section to match your database.
+- Replace every instance of `DSGRID_SS_REPO_BASE_PATH` with the path to your local
```

Comment:
Nit: Could this path be in just one place?

Reply:
JSON doesn't support references. The JSON format is tied to a dsgrid data model that has no requirements about common base directories. It's just one global find/replace in your editor.